xml/ay_custom_user_scripts.xml

```diff
@@ -402,14 +405,14 @@ echo "Testing the pre script" &gt; /tmp/pre-script_out.txt
         <entry>
          <para>
           Specify the interpreter that must be used for the script.
-          Supported options are <literal>shell</literal> and
-          <literal>perl</literal>.
+          Any interpreter available at given environment is possible
+          to specify. Full path even with parameters is possible.
```

Comment:
This needs to be re-phrased.

I would suggest:

"Any interpreter available in the given environment can be specified. It is possible to provide a full path to the interpreter, including parameters."

Reply:
sounds good.